.github/workflows/ruby.yml

```diff
@@ -0,0 +1,56 @@
+name: Ruby
+
+on:
+  push:
+    branches: [ develop ]
+  pull_request:
+    branches: [ develop ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        ruby-version: 
+          - 2.4
+          - 2.6
+          - 2.7
+          - 3.0
```

Comment:
```suggestion
          - '3.0'
```

This is a very small change, but which avoids a Float-to-String loss of characters.

An example (not from this project) of what it can look like before this change, in this picture:

<img width="376" alt="bild" src="https://user-images.githubusercontent.com/211/119782378-6ad6cd00-becc-11eb-9e13-9a425adaa054.png">

Read more details, if you like: https://github.com/actions/runner/issues/849

Reply:
(It applies everywhere a YAML float ending in .0 is used.)

Reply:
Thanks 👍